packages/core/src/lib/register.ts

```diff
@@ -169,17 +153,11 @@ export const registerWithPhoneAndPasscode = async (
 export const registerWithSocial = async (
   ctx: WithUserLogContext<Context>,
   provider: Provider,
-  { connectorId, code }: { connectorId: string; code: string }
+  { connectorId, result }: { connectorId: string; result: InteractionResults }
 ) => {
-  const userInfo = await getUserInfoByConnectorCode(connectorId, code);
+  const userInfo = await getUserInfoFromInteractionResult(connectorId, result);
```

Comment:
would move `getUserInfoFromInteractionResult` to the upper scope better?  `registerWithSocial` specifically takes userInfo as its input type. Just feels like passing in the `interactionResult` into a `registerWithSocial` method is not clean enough.

Reply:
updated